### ISSUE TITLE
Increase the touch and mouse area of scrubber.

### ISSUE THREAD

**Issue by veillette** [phetsims/calculus-grapher#230]
While looking at #207 with https://phet-dev.colorado.edu/html/calculus-grapher/1.0.0-dev.11/phet/calculus-grapher_all_phet.html
I mistakenly triggered the draglistener on the curve, when I was wishing to move the scrubber. 
The scrubber should have a larger touch and mouse area. However it should not be so large as to prevent curve manipulations.

**Comment by veillette** [comment]
The ScrubberNode adds a `XDragHandleNode` by composition.

The `XDragHandleNode` already dilates the touch area, but not the mouse area
``
    // Dilate the touch area a bit.
    options.touchArea = Shape.circle( 0, 0, options.radius + 5 );
``

**Comment by pixelzoom** [comment]
We have to be careful with increasing scrubber pointer areas because of this situation:

<img width="1074" alt="screenshot_2298" src="https://user-images.githubusercontent.com/3046552/218162559-4f2cb8aa-b62b-42b5-ae52-238f923fe13b.png">

**Comment by veillette** [comment]
I see that the touch area of the scrubber and reference line can even overlap. 

![image](https://user-images.githubusercontent.com/7633344/218164496-3d9614f1-5dc9-40ab-a951-220ef5674626.png)

Although, in practice, it doesn't seem problematic as 
- This occurs only on the second screen
- This occurs only when the scrubber and the reference line share the same x-value
- If the two draggable objects are closer together, a user would instinctively know that they need to be more precise in order to drag the desired object. 

It does seem more problematic though to trigger the curve, when one wanted to move the scrubber.

**Comment by veillette** [comment]
Testing a touch device, I think the touch area works fine. 
Hopefully we can find a similar sweet spot for the mouse area.

**Comment by veillette** [comment]
I added some mouseArea xDragHandleNode above

```js
    // Dilate the pointer areas a bit. (see https://github.com/phetsims/calculus-grapher/issues/230)
    options.touchArea = Shape.circle( 0, 0, options.radius + 5 );
    options.mouseArea = Shape.circle( 0, 0, options.radius + 2 );
```

There is very limited space for increase as we don't want to have overlap between the referenceLine and the scrubberNode. I think the mouse is an inherently more precise pointing device than touch and it would be startling hover over one and drag the other. 

In any case, the mouseArea of the scrubberNode should not be too large to prevent interactions with the curve.  

I attached an image of the pointerAreas.
![image](https://user-images.githubusercontent.com/7633344/218174892-ac14228c-8171-44e3-ad28-5ce6e88a541f.png)


Assigning to @pixelzoom to review.
